rpms/openshift-odo.spec

```diff
@@ -32,24 +30,31 @@ OpenShift Do (odo) is a fast, iterative, and straightforward CLI tool for develo
 
 %prep
 %setup -q -n %{source_dir}
+%patch0 -p1
```

Comment:
Where exactly is this patch coming from?

Reply:
Sorry, that was left over from earlier testing, fixed.